app/models/labware_creators/cardinal_pools_plate.rb

```diff
@@ -0,0 +1,139 @@
+# frozen_string_literal: true
+
+# TODO
+# Update presenter to display pools, and number of samples in each pool, for a destination well
+# Generate CSV
+
+# require_dependency 'form'
+# require_dependency 'labware_creators'
+
+# Steps:
+# 1. Create a new empty LCA PBMC Pools plate
+# 2. Get the passed wells from the parent (LCA PBMC) plate
+# 3. For the number of passed wells, get the number of pools from config
+# e.g. if there are 96 passed wells on the parent, the samples get split into 8 pools, with 12 samples per pool
+# 4. Group samples by supplier, to ensure samples with the same supplier are distrubuted across different pools
+# 5. Add the pool to a well in the new LCA PBMC Pools plate
+module LabwareCreators
+  # This class is used for creating randomicardinal pools into destination plate
+  class CardinalPoolsPlate < Base
+    include SupportParent::PlateOnly
+    # include LabwareCreators::CustomPage
+
+    def filters=(filter_parameters)
+      well_filter.assign_attributes(filter_parameters)
+    end
+
+    # This should only be called from passed_parent_wells
+    # As we want to only transfer passed wells to the LCA PBMC plate
+    def labware_wells
+      parent.wells
+    end
+
+    # returns: a list of passed samples
+    def passed_parent_wells
+      labware_wells.select { |well| well.state == 'passed' }
+    end
+
+    # returns: the number of pools required for a given passed samples count
+    # this config is appended in the Cardinal initialiser
+    # e.g. 95,12,12,12,12,12,12,12,11 ==> 8
+    # e.g. 53,11,11,11,10,10,,, ==> 5
+    def number_of_pools
+      Rails.application.config.cardinal_pooling_config[passed_parent_wells.count]
+    end
+
+    def well_filter
+      @well_filter ||= WellFilter.new(creator: self)
+    end
+
+    # Send the transfer request to SS
+    def transfer_material_from_parent!(dest_uuid)
+      dest_plate = Sequencescape::Api::V2::Plate.find_by(uuid: dest_uuid)
+      api.transfer_request_collection.create!(
```

Comment:
Do we want to handle if this request to create TransferRequests fails in some way?

Reply:
Add a test to show how the exception is handled